test/unit/lib/static/components/modals/screenshot-accepter/index.jsx

```diff
@@ -328,29 +285,38 @@ describe('<ScreenshotAccepter/>', () => {
 
         beforeEach(() => {
             const tree = mkEmptyTree();
-            addSuiteToTree({tree, suiteName: 'test-1'});
-            addBrowserToTree({tree, suiteName: 'test-1', browserName: 'bro-1'});
-            addResultToTree({tree, suiteName: 'test-1', browserName: 'bro-1', attempt: 0});
+            const suite = mkSuiteEntityLeaf('test-1');
+            addSuiteToTree({tree, suite});
+            const browser = mkBrowserEntity('bro-1', {parentId: suite.id});
+            addBrowserToTree({tree, browser});
+            const result = mkResultEntity('res-1', {parentId: browser.id});
+            addResultToTree({tree, result});
+
             for (let i = 1; i <= 10; i++) {
-                addImageToTree({
-                    tree,
-                    suiteName: 'test-1',
-                    browserName: 'bro-1',
-                    attempt: 0,
-                    stateName: `state-${i}`,
-                    expectedImgPath: `img${i}-expected.png`,
-                    diffImgPath: `img${i}-diff.png`,
-                    actualImgPath: `img${i}-actual.png`
-                });
+                const image = mkImageEntityFail(`state-${i}`, {parentId: result.id});
+                addImageToTree({tree, image});
+
+                // addImageToTree({
+                //
+                //     tree,
+                //     suiteName: 'test-1',
+                //     browserName: 'bro-1',
+                //     attempt: 0,
+                //     stateName: `state-${i}`,
+                //     expectedImgPath: `img${i}-expected.png`,
+                //     diffImgPath: `img${i}-diff.png`,
+                //     actualImgPath: `img${i}-actual.png`
+                // });
             }
             const store = mkRealStore({initialState: {tree, view: {expand: EXPAND_ALL}}});
 
-            const currentImageId = generateImageId({
-                suiteName: 'test-1',
-                browserName: 'bro-1',
-                attempt: 0,
-                stateName: 'state-5'
-            });
+            const currentImageId = Object.values(tree.images.byId)[4].id;
+            // generateImageId({
+            //     suiteName: 'test-1',
+            //     browserName: 'bro-1',
+            //     attempt: 0,
+            //     stateName: 'state-5'
+            // });
```

Comment:
Looks like you should check check if there is such a commented-out code in your PR that can be deleted